Scholars Portal Journals.js

```diff
@@ -0,0 +1,172 @@
+{
+	"translatorID": "5ac0fd37-5578-4f82-8340-0e135b6336ee",
+	"label": "Scholars Portal Journals",
+	"creator": "Bartek Kawula",
+	"target": "https?://journals[1-2]\\.scholarsportal\\.info/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-02-19 04:18:37"
+}
+
+function detectWeb(doc, url) { 
+	// see if saved list is toggled open
+	if (doc.getElementsByClassName('inner-wrap-open') [0]) {
```

Comment:
This will need to be a bit more involved. Since the page is not being loaded, detectWeb is not triggered when the list is shown/hidden. You will want to use `Zotero.monitorDOMChanges`. For an example, you can take a look at a couple translators [here](https://github.com/zotero/translators/search?utf8=%E2%9C%93&q=monitorDomChanges) (some are more complicated than others). In this case, I think you would want something like this at the beginning of detectWeb:
```
var myArticles = doc.getElementById('my_articles');
if (myArticles) {
  Zotero.monitorDOMChanges(myArticles.parentElement, {attributes: true, attributeFilter: ["style"]});
}
```